src/core/processing/models/qgsprocessingmodelalgorithm.cpp

```diff
@@ -769,7 +769,7 @@ QStringList QgsProcessingModelAlgorithm::asPythonCode( const QgsProcessing::Pyth
     const QString base = safeName( name, capitalize );
     QString candidate = base;
     int i = 1;
-    while ( friendlyNames.contains( candidate ) )
+    while ( std::find( friendlyNames.cbegin(), friendlyNames.cend(), candidate ) != friendlyNames.cend() )
```

Comment:
what's the practical difference here? It looks logically identical to me.

Reply:
That's pretty subtle, can you add a comment?

Reply:
Done